### ISSUE TITLE
Program exits when compiling openclxxx.bin with AMD Radeon drivers version 19.7.1 

### ISSUE THREAD

**Issue by ephebus** [fireice-uk/xmr-stak#2473]
After installing the latest AMD Radeon drivers - version 19.7.1, released on 7 July 2019 - xmr-stak 2.10.5 starts compiling openclxxx.bin and then exits. No new .bin file is created in the program's data directory. Tried it with Monero and Ryo. Had to revert to the previous drivers version (19.6.3). Windows 7 x64 SP1, RX 570 4GB.

**Comment by psychocrypt** [comment]
Do you stored the output of the miner. I am interested into the logs.

**Comment by ephebus** [comment]
I don't think so. What should I add to the start line to save the logs you need? I can install the 19.7.1 drivers again and get them for you.

**Comment by ephebus** [comment]
OK, so I edited the config.txt file and set verbose_level to 10, enabled the log file and reinstalled the 19.7.1 drivers. When starting xmr-stak to mine Ryo, this is all that gets logged:

xmr-stak 2.10.5 b846a6f7c
(...)
[2019-07-10 02:18:50] : Mining coin: cryptonight_gpu
[2019-07-10 02:18:50] : Compiling code and initializing GPUs. This will take a while...
[2019-07-10 02:18:50] : Number of OpenCL GPUs 1
[2019-07-10 02:18:50] : OpenCL Init device 0
[2019-07-10 02:18:50] : Device 0 work size 16 / 16.
[2019-07-10 02:18:51] : OpenCL device 0 - Precompiled code C:\Users\xxx\AppData\Local/.openclcache/9fb790f12ed12a10d955a1a9b130ea4270abf836ba131ef558b123a9d2da7490.openclbin not found. Compiling ...

For Monero:

[2019-07-10 02:25:23] : CryptonightR create random math for block 1806260
-------------------------------------------------------------------
xmr-stak 2.10.5 b846a6f7c
(...)
[2019-07-10 02:25:23] : Mining coin: cryptonight_r
[2019-07-10 02:25:23] : Compiling code and initializing GPUs. This will take a while...
[2019-07-10 02:25:23] : Number of OpenCL GPUs 1
[2019-07-10 02:25:23] : OpenCL Init device 0
[2019-07-10 02:25:23] : Device 0 work size 8 / 32.
[2019-07-10 02:25:24] : OpenCL device 0 - Precompiled code C:\Users\xxx\AppData\Local/.openclcache/ee289a7f8d491cbfc337f7ba025fd6863a54c153a9c554065f8339ca24620c2d.openclbin not found. Compiling ...

**Comment by psychocrypt** [comment]
You can activate log files in confic.txt. But coping from the terminal is
also ok.

**Comment by ephebus** [comment]
Yes, I just thought there might be something else that gave a more detailed output. I've posted all that gets logged above.

**Comment by alLe746** [comment]
got exactly the same issue with new drivers.

**Comment by kybernetyk** [comment]
just to chime in: xmrig-amd also segfaults since the last driver update (under arch linux). so that problem seems to be with the drivers

**Comment by trg100** [comment]
Also have this issue:
[2019-07-11 20:56:50] : Mining coin: cryptonight_r
[2019-07-11 20:56:50] : Compiling code and initializing GPUs. This will take a while...
[2019-07-11 20:56:50] : Device 0 work size 8 / 32.
[2019-07-11 20:56:50] : OpenCL device 0 - Precompiled code /home/tom/.openclcache/63cae9eaa0071713225f4208471d984d34ba18bba463d29731a769d8fc5191c0.openclbin not found. Compiling ...
Segmentation fault (core dumped)

**Comment by Brun0F** [comment]
Problem persists with 19.7.2 drivers.

**Comment by Sonic2008** [comment]
I'm having the same issue, maybe we should roll back the driver? Possibly 
![Capture](https://user-images.githubusercontent.com/33471823/61493388-73315900-a981-11e9-9383-5f6da8f3b439.JPG)

**Comment by psychocrypt** [comment]
currently yes you must role back. I need to check whats going on

Sonic2008 <notifications@github.com> schrieb am Do., 18. Juli 2019, 23:28:

**Comment by draggy** [comment]
I don't know if it's related but the linux builds do the same thing with kernel 5.2.1, Vega 64, amdgpu driver

**Comment by axi6ne8us** [comment]
Install this program a few minutes ago and getting this same error. My system has a Radeon VII running version 19.7.2

**Comment by psychocrypt** [comment]
fixed with #2482, will be released today or tomorrow.
